pages/index.js

```diff
@@ -31,6 +31,19 @@ export async function getServerSideProps(context) {
   };
 }
 
+//will change the location value in the selector to be whatever the geolocation API returns
+//also returns the canadian province or "Other" the api acquires
+async function assumeLocation() {
```

Comment:
I'd rename to something along the lines of getUserLocation() to make the intention clear